__tests__/app.test.js

```diff
@@ -5,4 +5,48 @@ const { string } = require("pg-format");
 
 afterAll(() => {
   if (db.end) db.end();
+});
+
+describe('GET /api', () => {
```

Comment:
there should be no testing for GET /api right now, this comes later in the project and has a very specific response. Please remove this for not